src/components/Checkbox/Checkbox.st.css

```diff
@@ -52,10 +52,30 @@
 
 .root::childContainer {
     display: flex;
+    flex: 1;
+    flex-wrap: wrap;
+    align-items: center;
 }
 
 .root::box {
-    line-height: 20px;
+    line-height: 0;
+    margin-right: 12px;
```

Comment:
Note that we need to support `rtl` preferably out-of-the-box.
So instead of using `margin-right` and such, we prefer to us `::after` and give it the `width` we expect the `margin` to be.